refract-spec.md

```diff
@@ -498,32 +480,34 @@ The following shows a link with the relation of `foo` and the URL of `/bar`.
 }
 ```
 
-## Namespacing
+## Profiles
 
-Namespaces provide a way to include element definitions and constraints into another document. The following rules and constraints apply to namespaces.
+The primary means by which users can provide semantic definitions and other meta information is through a profile. A profile MAY provide semantic information about an element and its data, it MAY provide specific instructions about elements such as how inheritance should work or how elements should be processed, and it MAY be used to modify understanding of existing elements in other profiles. The usage is a profile is not limited to these descriptions, and SHOULD be left up to the profile author to define its use.
 
-1. If there is a conflict, the namespaced element defined or referenced last MUST take precedence
-1. Types defined within the local namespace MUST take precedence over referenced namespaces
-1. Prefixed namespaces are only accessible through using a prefix
-1. Namespaces apply to the element in which they are introduced and all child elements
-1. Namespaces MUST not include the elements nor the data in the elements referenced, but rather include the constraints and conditions around the types defined within that namespace
+To point to a profile, you MAY use the [profile link relation](https://www.ietf.org/rfc/rfc6906.txt) as a meta link in your root element or in any other element. Profile links may also be found outside of the document itself in places like the [HTTP Link Header](http://www.w3.org/wiki/LinkHeader).
```

Comment:
Is it necessary to point to the profile when an element from that profile is used?